Storage/Storage/SQL/BrowserDB.swift

```diff
@@ -8,11 +8,22 @@ public enum QuerySort {
     case None, LastVisit, Frecency
 }
 
+public class DateRange {
```

Comment:
This class is the ideal candidate for a struct i think. Because it is pretty much a read-only value type.